app/javascript/css/forms.scss

```diff
@@ -127,7 +127,7 @@ fieldset label:not(:last-child) {
   margin-right: $spacer;
 }
 
-label.radio {
+label.radio, label.unweighted, legend.unweighted {
```

Comment:
I think we already have a class for this font weight that Kristin and I have been using: 'weight-n'. 
Give that a go, but if it doesn't cooperate then this is a fine solution.

Reply:
oh nice, just tried it out and it does indeed work